firmware/util/timer.cpp

```diff
@@ -2,18 +2,10 @@
 
 #include "timer.h"
 
-Timer::Timer() {
-	init();
-}
-
 void Timer::reset() {
 	m_lastReset = getTimeNowNt();
 }
 
-void Timer::init() {
-	m_lastReset = INT64_MIN / 8;
```

Comment:
i believe that this API is needed regardless, at the moment we cannot get timer back into original state